workers/multi-globals/entry.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Multiple globals for Worker constructor: parsing scriptURL</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<iframe src="incumbent/incumbent.html"></iframe>
+<button onclick="frames[0].hello()" id="button">Hello</button>
+
+<script>
+async_test((t) => {
+  onload = () => {
+    document.getElementById('button').click();
```

Comment:
The indirection through the button click shouldn't be necessary. Compare to e.g.

- https://github.com/web-platform-tests/wpt/blob/master/fetch/api/request/multi-globals/url-parsing.html
- https://github.com/web-platform-tests/wpt/blob/master/webmessaging/multi-globals/messageport-current.html

Reply:
Fixed